apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -351,6 +356,38 @@ export class UpsertWorkflowUseCase {
     const newControlValues = controlValues || {};
     if (step.template?.type === StepTypeEnum.EMAIL) {
       const emailControlValues = newControlValues as EmailControlType;
+
+      const isLayoutsPageActive = await this.featureFlagsService.getFlag({
+        key: FeatureFlagsKeysEnum.IS_LAYOUTS_PAGE_ACTIVE,
+        defaultValue: false,
+        environment: { _id: command.user.environmentId },
+        organization: { _id: command.user.organizationId },
+      });
+
+      // Assign default layoutId if undefined (but not if null)
+      if (isLayoutsPageActive && emailControlValues.layoutId === undefined) {
```

Comment:
do only when ff is on; when the email step control value `layoutId` is undefined assign the default layout `layoutId` value, otherwise validate what used provides and assign it (null or value).